lib/puppet/type/rds_instance.rb

```diff
@@ -196,6 +198,17 @@ def insync?(is)
     end
   end
 
+  newproperty(:backup_retention_period) do
+    desc 'The number of days backups should be retained for.'
```

Comment:
Please use the same wording as the README. "The number of days to retain backups. Defaults to 30 days." just reads much better.